requirements.txt

```diff
@@ -22,3 +22,4 @@ six==1.15.0
 shodan==1.25.0
 pycurl==7.43.0.5
 xmltodict==0.12.0
+icmplib
```

Comment:
mention condition to install icmplib if it using pip3 .

Reply:
for `attack.py` and `engine.py` you should add condition that if user use python3 then this code run otherwise default old code should run(with root privileges). This will not break the program if user run this using python2.